PhysicsTools/NanoAOD/python/taus_cff.py

```diff
@@ -160,7 +166,18 @@ def _tauId8WPMask(pattern,doc):
 run2_miniAOD_80XLegacy.toModify(tauTable,
                                 variables = _variables80X
 )
+(~run2_miniAOD_devel).toModify(tauTable.variables,
+                               idAntiEleDeadECal = None
+)
```

Comment:
@mbluj 
what about to recompute 'idAntiEleDeadECal' for the old version of mini input ?
we have done in master and for the ID below you recompute all such that you get always filled with something

Reply:
In my understanding in `master` it is desirable to have "latest and greatest" nanoAOD content regardless of input miniAOD / era, while for older release series nanoAOD content is frozen with old eras to allow reproducibility (no changing policy as for other data-tiers). This implementation follows above reasoning. It can be however changed to be closer to master if it is expected behaviur.

Reply:
for reco and miniAOD holds what you said " (no changing policy as for other data-tiers)"
But nanoAOD are a simple ntuplizer.
We want to have the fix and latest greatest variable compatible with the input dataset and the corrections/calibration you have available for the analysis. So we can update the content as we update the pre-releases.

So worth to add the idAntiEleDeadECal for the EOY and UL106v1.
Also this is a new variable, is someone want to put together with other ntuples where you didn't have one can just ignore.

@peruzzim